app/src/main/java/org/jellyfin/androidtv/ui/NowPlayingBug.java

```diff
@@ -125,7 +123,7 @@ private void setInfo(BaseItemDto item) {
         if (item == null) return;
 
         Glide.with(context)
-                .load(ImageUtils.getPrimaryImageUrl(item, KoinJavaComponent.<ApiClient>get(ApiClient.class)))
+                .load(ImageUtils.getPrimaryImageUrl(item))
```

Comment:
this image didn't load before this PR, and it still doesn't

Reply:
This one works for me, I think it requires a primary image for the track (not the album) to show up.